### PR TITLE
DLS-6211: [RP] Session timeout dialog.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -56,20 +57,33 @@
...
 
-
-
 @pageScripts = {
   @scriptElem
   <script src='@routes.Assets.versioned("javascripts/show-hide-content.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/do-not-track.js")'></script>
+  <script>
+      window.jsConfig = window.jsConfig || {};
+      window.jsConfig.timeoutEnabled = @if(isSignedIn){ true } else {false};
```
Comment (nubz) [inline app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html:66]: can this not be simplified to `window.jsConfig.timeoutEnabled = @{isSignedIn};`

Comment (nubz) [reply app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html:66]: in fact why can we not use `enableSignOutLink` which looks like it already exists to test signed in status?

Comment (rambabu-posa) [reply app/uk/gov/hmrc/helptosavefrontend/views/govuk_wrapper.scala.html]: @nubz yes right fixed, please review it.

File: app/uk/gov/hmrc/helptosavefrontend/views/main_template.scala.html [hunk 2]
```diff
@@ -60,5 +60,6 @@
   mainContentHtml = article(mainContent),
   scriptElem = scriptElem,
   extraMeta = extraMeta,
-  enableSignOutLink = enableSignOutLink
+  enableSignOutLink = enableSignOutLink,
+  isSignedIn = isSignedIn
```
Comment (nubz) [inline app/uk/gov/hmrc/helptosavefrontend/views/main_template.scala.html:64]: this should not be here now

File: conf/messages [hunk 3]
```diff
@@ -796,6 +796,19 @@ hts.reminder-apply-savings=Do you want to receive email savings reminders?
...
+hts.timeout.timeoutSignOutButtonText = Sign out
+hts.timeout.timeoutMinutes = minutes
+hts.timeout.timeoutMinute = minute
+hts.timeout.timeoutSeconds = seconds
+hts.timeout.timeoutSecond = second
+
+hts.timed-out.p1=This was because you were inactive for too long.
+hts.timed-out.p2=To access your Help to Save account, you’ll need to sign in.
+hts.timed-out.button=Sign in
+
```
Comment (nubz) [inline conf/messages:811]: there doesn't seem to be any Welsh for the last 3 messages

File: conf/messages.cy [hunk 4]
```diff
@@ -795,6 +795,15 @@ hts.reminder-apply-savings=A ydych am gael e-byst atgoffa ynghylch cynilion?
...
 
+hts.timeout.timeoutTitle = Rydych ar fin cael eich allgofnodi
+hts.timeout.timeoutMessage = Er eich diogelwch, byddwn yn eich allgofnodi cyn pen
+hts.timeout.timeoutKeepAliveButtonText = Parhau i fod wedi’ch mewngofnodi
+hts.timeout.timeoutSignOutButtonText = Allgofnodi
+hts.timeout.timeoutMinutes = funud
+hts.timeout.timeoutMinute = funud
+hts.timeout.timeoutSeconds = eiliad
+hts.timeout.timeoutSecond = eiliad
+
```
Comment (nubz) [inline conf/messages.cy:806]: are we waiting for Welsh translations on the missing messages?

**Review by nubz — APPROVED** [review]
LGTM

**Pull request merged**